packages/button-react/example/index.tsx

```diff
@@ -1,23 +1,43 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import { PrimaryButton, SecondaryButton, TertiaryButton } from "../src";
+import { initTabListener } from "@fremtind/jkl-core";
 import "@fremtind/jkl-button/button.css";
-import "@fremtind/jkl-core/core.min.css";
+import "@fremtind/jkl-core/core.css";
+
+initTabListener();
 
 function onClick() {
     console.log("Hello!");
 }
 
 const Buttons = () => (
     <>
-        <div style={{ margin: "20px" }}>
-            <PrimaryButton onClick={onClick}>Hello, Primary Button</PrimaryButton>
+        <div className="side-by-side">
+            <pre>
+                <code>{`forceCompact={false}`}</code>
+            </pre>
+            <pre>
+                <code>{`forceCompact={true}`}</code>
+            </pre>
+        </div>
+        <div className="side-by-side">
+            <PrimaryButton onClick={onClick}>Send inn</PrimaryButton>
+            <PrimaryButton forceCompact onClick={onClick}>
+                Send inn
+            </PrimaryButton>
         </div>
-        <div style={{ margin: "20px" }}>
-            <SecondaryButton onClick={onClick}>Hello, Secondary Button</SecondaryButton>
+        <div className="side-by-side">
+            <SecondaryButton onClick={onClick}>Svar senere</SecondaryButton>
+            <SecondaryButton forceCompact onClick={onClick}>
+                Svar senere
+            </SecondaryButton>
         </div>
-        <div style={{ margin: "20px" }}>
-            <TertiaryButton onClick={onClick}>Tertiary</TertiaryButton>
+        <div className="side-by-side">
+            <TertiaryButton onClick={onClick}>Avbryt</TertiaryButton>
+            <TertiaryButton forceCompact onClick={onClick}>
+                Avbryt
+            </TertiaryButton>
```

Comment:
If you view this on a mobile-device, it looks the same. That might not be an issue, but the demo kinda don't work in that case.

Is there any case, besides a demo like this to force into large?

Reply:
Yeah, I see your point! It does still demonstrate the functionality, but it's impossible to see the difference 🤔 It looks like we're heading towards keeping the big size on mobile, so I see no point in implementing a `forceLarge` functionality per now…

Reply:
For sure, we should not implement forceLarge just for test/demo purposes.